DIGNDB.App.SmitteStop.Core/Helpers/ExposureBatchFileUtil.cs

```diff
@@ -54,16 +54,16 @@ private string GetPrivateKeyFromCertificate(string certificateThumbprintOrPath)
             }
         }
 
-        public SignatureInfo[] Signatures => new SignatureInfo[]
+        public SignatureInfo[] Signatures => new []
         {
-    				new SignatureInfo
-    				{
-      					VerificationKeyId = "238",
-      					VerificationKeyVersion = "v1",
-      					SignatureAlgorithm = "1.2.840.10045.4.3.2",
-      					PrivateKey = _privateKey
-    				},
-  		};
+            new SignatureInfo
+            {
+                VerificationKeyId = "242",
```

Comment:
Sorry for bothering but could you create a named const for it?
Like
`const string NorwegianVerificationKeyId = "242";` and use it in `new SignatureInfo`?

Reply:
It defined i proto as far as I can tell and autogenerated. Let's take a closer look at it later. For now we just need this value to be changed.

Reply:
Okay!